omx/gstomx.c

```diff
@@ -1251,13 +1248,14 @@ gst_omx_port_compare_buffers (gconstpointer a, gconstpointer b)
  * If *buf is not NULL the buffer containing the given omx pointer
  * is going to be returned */
 GstOMXAcquireBufferReturn
-gst_omx_port_acquire_buffer (GstOMXPort * port, GstOMXBuffer ** buf)
+gst_omx_port_acquire_buffer (GstOMXPort * port, GstOMXBuffer ** buf, gboolean retry_limit)
```

Comment:
Add retry_limit explanation/comments somewhere in the function.

Reply:
Added in commit 577747aa1d7afe875676172b923ad778d8ccd06b